ats/src/ats/util/base.py

```diff
@@ -36,21 +36,19 @@ def __setPositionalFields(self, args):
         nameIter = iter(self._fieldNames)
         for value in args:
             try:
-                name = nameIter.next()
+                name = next(nameIter)
                 self.__setattr__(name, value)
             except StopIteration:
-                raise AttributeError, \
-                    'Too many positional args initialzing %s object.  Expected %s, but got %s.' %\
-                    (self.__class__, len(self._fieldNames), len(args))
+                raise AttributeError('Too many positional args initialzing %s object.  Expected %s, but got %s.' %\
```

Comment:
(not sure) do we still need this `\`? I think because the `AttributeError` args are in parens we don't?

Reply:
That is correct, the ending `\` is not needed. I'll drop this in the next commit.